java_console/io/src/main/java/com/rusefi/SimulatorExecHelper.java

```diff
@@ -93,9 +94,27 @@ static void destroy() {
     }
 
     public static void startSimulator() {
-        if (!new File(SIMULATOR_BINARY).exists())
-            throw new IllegalStateException(SIMULATOR_BINARY + " not found");
+        getSimulatorBinary();
+
         FileLog.MAIN.logLine("startSimulator...");
         new Thread(SimulatorExecHelper::runSimulator, "simulator process").start();
     }
+
+    private static File getSimulatorBinary() {
+        return getSimulatorBinary(SIMULATOR_BINARY_PATH + SIMULATOR_BINARY_NAME);
```

Comment:
is it worth requiring passing `SIMULATOR_BINARY_NAME` every time to `getSimulatorBinary`? Seems redundant.